dispatcher_test.go

```diff
@@ -295,3 +295,14 @@ func TestStartStopFailures(t *testing.T) {
 		}
 	}
 }
+
+func TestNoOutboundsForService(t *testing.T) {
+	assert.Panics(t, func() {
```

Comment:
nit - I would test the error message as well. For panic, you might need to see if the stack contains the error message, instead of equaling.